Sources/PaymentButtons/PaymentButtonColor.swift

```diff
@@ -29,28 +29,35 @@ public enum PaymentButtonColor: String {
             return UIColor(hexString: "#FFFFFF")
 
         case .black:
-            return UIColor(hexString: "#000000")
+            return UIColor(hexString: "#FFFFFF")
 
         case .silver:
-            return UIColor(hexString: "#EEEEEE")
+            return UIColor(hexString: "#FFFFFF")
 
         case .blue:
-            return UIColor(hexString: "#0070BA")
+            return UIColor(hexString: "#FFFFFF")
 
         case .darkBlue:
-            return UIColor(hexString: "#073990")
+            return UIColor(hexString: "#FFFFFF")
         }
     }
 
     var fontColor: UIColor {
```

Comment:
Same here, considering all _cases_ have the same _fontColor_ value

```suggestion
    var fontColor: UIColor {
        UIColor(hexString: "#001435")
    }
```